datalad/support/external_versions.py

```diff
@@ -252,6 +255,43 @@ def keys(self):
     def __contains__(self, item):
         return bool(self[item])
 
+    def add(self, name, func=None, interesting=True):
+        """Add a version checker
+
+        Could be used e.g. by extensions to add important to them custom checks.
+        If will not add `name` if already exists.  If it exists and `func` is
+        different - it will override with a new `func`.
+
+        Parameters
+        ----------
+        name: str
+          Name of the check (usually a name of the Python module, or an
+          external command prefixed with "cmd:")
+        func: callable, optional
+          Function to be called to obtain version information, if it is not
+          a Python module following standard ways to expose version via
+          `__version__`
+        interesting: bool, optional
+          Makes sense only for entries without func - either it is considered
+          "interesting" and should be included in output of dumps
```

Comment:
Is this needed?  Would anyone use this method with something they thought wasn't interesting?

Reply:
well, that is a bigger question I guess. Before now, all the `INTERESTING` was just an attribute which nobody care about. Here I just thought to make sure to describe what effect it would have to have something "interesting" (that only then `dumps` would take care to query it if it was not queried).

Reply:
Do you anticipate any caller of this method will want to add something and call `interesting=False`?  If not, I'd say this is unnecessary cruft.  I don't think it should exist just for the purpose of documenting `.INTERESTING`, which can be documented in more usual ways.

Reply:
hm... ok -- you think I should just get rid of this option, and add to `INTERESTING` whenever there is no `func` provided?

Reply:
> you think I should just get rid of this option, and add to `INTERESTING` whenever there is no `func` provided?

yes

Reply:
ok, did in 83d010ecb361ff51658c93a0ef990f74ac8e894e